### PR TITLE
minor change to help eyesight on instructions

### LINKED ISSUE #2: White space in instructions
(partly testing filing an issue)

The instructions are hard to follow mostly as you flick from them to other files - some whitespace between the bullets would help.  Many other things would help too, but this is a simple start.

### PR DESCRIPTION
**Pull request by iainwp**
Hi, Tom. Partly helping my eyes, partly testing pull requests - feel free to ignore.

### CODE REVIEW

File: WizardStyle.css [hunk 1]
```diff
@@ -41,3 +41,11 @@ details {
 summary, .setAll {
   background-color: #F56600;
 }
+
+li {
+    margin-bottom: 2ex;
+}
+
+#vbutton {
```
Comment (tdobra) [inline WizardStyle.css]: I cannot find the element with ID vbutton. Did you mean to add this to the CSS?

File: wizard.html [hunk 2]
```diff
@@ -36,9 +36,10 @@ <h1>TCTemplate: TrailO Timed Control Map Cards Generator</h1>
...
+<p>Upload the <strong>maps</strong> PDF: <input type="file" id="coursePDFSelector" accept=".pdf" /></p></li>
+        <li>Create a single PDF of all the control descriptions using File->Create PDFs->Descriptions. Choose fill one page with multiple copies of each course. Select all courses, including those not intended to be printed, but not the all controls. Choose a PDF printer with A4 page size. Leave the box size at the default of 7 mm, and leave the page margins at the default of 12.7 mm on all sides (appears as 13 mm in the main dialog).<p>Upload the <strong>descriptions</strong> PDF: <input type="file" id="CDPDFSelector" accept=".pdf" /></p></li>
+        <li>Save the Purple Pen file, then <p>Upload the <strong>purple pen file </strong> to this wizard: <input type="file" id="ppenSelector" accept=".ppen" onchange="tcTemplate.loadppen(this)" /><input type="button" value="Reload" onclick="tcTemplate.loadppen(document.getElementById('ppenSelector'))" /></p></li>
```
Comment (tdobra) [inline wizard.html]: Capital letters required on Purple Pen. For consistency, only make Purple Pen and not file bold. Also, break the sentence after save file.

File: wizard.html [hunk 3]
```diff
@@ -36,9 +36,10 @@ <h1>TCTemplate: TrailO Timed Control Map Cards Generator</h1>
...
-        <li>Save the Purple Pen file, then upload it to this wizard: <input type="file" id="ppenSelector" accept=".ppen" onchange="tcTemplate.loadppen(this)" /><input type="button" value="Reload" onclick="tcTemplate.loadppen(document.getElementById('ppenSelector'))" /></li>
+        <li>Create a single PDF of all the courses using File->Create PDFs->Courses and choosing to have one copy of each course all in a single PDF. Select all courses, including those not intended to be printed, but not the all controls map. Select <em>CMYK</em> as the colour model. 
+<p>Upload the <strong>maps</strong> PDF: <input type="file" id="coursePDFSelector" accept=".pdf" /></p></li>
```
Comment (tdobra) [inline wizard.html]: I don't like the style of a `<p>` for part of an `<li>`; use `<br />` instead.

File: WizardStyle.css [hunk 4]
```diff
@@ -41,3 +41,11 @@ details {
 summary, .setAll {
   background-color: #F56600;
 }
+
+li {
```
Comment (tdobra) [inline WizardStyle.css]: This change affects other pages on the site that look better without the extra spacing. Need to create a class to select the elements that need this spacing. Also, is there a way to apply this to the `<ol>` rather than each `<li>` to save an excessive number of style attributes?

**Review by tdobra — CHANGES_REQUESTED** [review]
I like the new formatting, but a few changes are required to make it work with the rest of the site.

File: WizardStyle.css (hunk 1 above)
Comment (iainwp) [reply WizardStyle.css]: At the time i added it I did, but got overtaken by events ... that bit can be removed.

File: wizard.html (hunk 3 above)
Comment (iainwp) [reply wizard.html]: up to you.  I don't like `<br/>` but it's your code.

File: WizardStyle.css (hunk 4 above)
Comment (iainwp) [reply WizardStyle.css]: Class is probably best

File: wizard.html (hunk 3 above)
Comment (tdobra) [reply wizard.html]: Also, the use of `<strong>` looks good here. As part of having a good house style, what do you think is best for describing settings? I have previously used `<em>` but I'm no longer convinced I have used the best tag for the job.

Comment (iainwp) [reply wizard.html]: I was finding very difficult to see the buttons. Then It was difficult to see which was which. In this font em doesn’t show up as much as strong.

File: wizard.html (hunk 2 above)
Comment (tdobra) [reply wizard.html]: Done and committed.

File: WizardStyle.css (hunk 4 above)
Comment (tdobra) [reply WizardStyle.css]: Setting line-height instead on the `<ol>` looks like it's worth a try.
